uk.co.compucorp.civicrm.hrleaveandabsences/js/src/leave-absences/leave-type-wizard/components/leave-type-wizard.component.js

```diff
@@ -16,16 +16,20 @@ define([
       controllerAs: 'form',
       templateUrl: ['shared-settings', function (sharedSettings) {
         return sharedSettings.sourcePath + 'leave-type-wizard/components/leave-type-wizard.html';
-      }]
+      }],
+      bindings: {
+        leaveTypeId: '@'
+      }
     }
   };
 
   function LeaveTypeWizardController ($log, $q, $scope, $window, AbsenceType,
-    Contact, customTabNamesByCategory, fieldsHiddenByCategory, formSections,
+    Contact, customTabNamesByCategory, dialog, fieldsHiddenByCategory, formSections,
     leaveTypeCategoriesIcons, hiddenTabsByCategory, notificationService,
     OptionGroup, overridesByCategory, sharedSettings) {
     $log.debug('Controller: LeaveTypeWizardController');
 
+    var absenceType;
```

Comment:
I think it would be good to name this property "originalAbsenceType" or similar, because it might get confused with the object that stores the fields that are being edited.

Reply:
Done.